sdk/ai/azure-ai-agents/tests/test_ai_agents_instrumentor.py

```diff
@@ -192,7 +192,8 @@ def test_agent_chat_with_tracing_content_recording_enabled(self, **kwargs):
         # delete agent and close client
         client.delete_agent(agent.id)
         print("Deleted agent")
-        messages = client.messages.list(thread_id=thread.id)
+        messages = [msg for msg in client.messages.list(thread_id=thread.id)]
```

Comment:
For sync iterator we could also do direct casting to list
list(client.messages.list(thread_id=thread.id)) 

But like this it is ok.